### DISCUSSION TITLE
SOCKS5 with username and password

### DISCUSSION THREAD

**Discussion by v-2841** [pear-devs/pear-desktop#2779] [Q&A]
Hi!
I can't set up proxy with username and password.
URL like this doesn't work: socks5://username:password@server_host:port
The problem not with server. Tested with: `curl --socks5 server_host:port -U username:password https://music.youtube.com/`
Any ideas?

**Comment by Daksh777** [comment]
Were you able to figure this out @v-2841? Same problem here
